src/main/java/com/recurly/v3/exception/ExceptionFactory.java

```diff
@@ -68,4 +69,40 @@
     }
     return (T) apiException;
   }
+
+  @SuppressWarnings("unchecked")
+  public static <T extends RecurlyException> T getExceptionClass(Response response) {
+    String requestId = response.header("X-Request-Id", "");
```

Comment:
Maybe we should change the default to `"null"` or `"none"`. What do you think @douglasmiller ?

Reply:
Yeah I think that's a good idea since it will be a part of the error message. Will update in the morning